#!/usr/bin/env python
PACKAGE = "auv_smach"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("selected_animal", str_t, 0, "Selected animal for mission (shark or sawfish)", "sawfish")
gen.add("wall_reference_yaw", double_t, 0, "Yaw angle of the wall reference frame", -2.5, -3.14, 3.14)
gen.add("slalom_direction", str_t, 0, "Slalom direction", "left")

exit(gen.generate(PACKAGE, "auv_smach", "SmachParameters"))
